src/modules/markAsComplete.js

```diff
@@ -0,0 +1,14 @@
+const markAsComplete = () => {
+  const markComplete = document.querySelectorAll('.check-box');
+  markComplete.forEach((element, index) => {
+    element.addEventListener('input', (event) => {
+      const items = JSON.parse(localStorage.getItem('toDoList')) || [];
+      items[index].completed = event.target.checked;
+      localStorage.setItem('toDoList', JSON.stringify(items));
+      // eslint-disable-next-line no-restricted-globals
+      location.reload(true);
```

Comment:
Please ensure that we do not disable eslint in our code base, we can fix it by placing the disable statement to the webpack bundlers and config files. Kindly remove eslint disable comments from the various files.

Reply:
Thanks for the review it's really a good practice to maintain eslint checks in my codebase and not disable them directly within the code. 💯

Reply:
> Thanks for the review it's really a good practice to maintain eslint checks in my codebase and not disable them directly within the code. 💯

You are welcome. Feel free to request a re-review when you are done implementing the changes 👌

Reply:
Hi @efmomoh 

For me to use eslint-disable-next-line comment is really necessary for my project to run smoothly I've tried other ways hope you can understand my dilemma 

below is a tip from my previous project

TIP: If you separated the code in multiple files and you have the following issues with the linter: (i) complaining about having multiple classes at the same file, or (ii) unused variables, disable the linter check for that rule in the file with a /* eslint-disable rule-you-want-to-disable */.

Thank you in advance

Reply:
Based on the specifications, a Linters check is required. That means if our project linters can not check that means there should be a major issue with our syntaxes and structure of files. However, you may esLint disable the particular line of code that is making complaints and push your changes thereby enabling linters to run on GitHub workflows. When you are done, ask for a re-review.